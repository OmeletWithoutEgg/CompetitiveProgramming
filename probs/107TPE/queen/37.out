182004.70 33469848.49
