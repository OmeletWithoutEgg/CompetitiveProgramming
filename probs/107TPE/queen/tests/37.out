-502153.21 -4844211.69
